File: python/8 kyu/389.py
# https://www.codewars.com/kata/56a946cd7bd95ccab2000055/train/python

# Passed

def lowercase_count(strng):
    result = sum(char.isalpha() and char.islower() for char in strng)
    return result

output = lowercase_count("abcABC123")
print(output)

File: nodejs/8 kyu/389.js
// https://www.codewars.com/kata/55685cd7ad70877c23000102/train/javascript

// Passed

function makeNegative(num) {
  let negativeNumber = num > 0 ? num * -1 : num;
  return negativeNumber;
}

const output = makeNegative(42);
console.log(output);